src/components/AuditLogsTable/AuditLogsTable.tsx

```diff
@@ -54,8 +50,9 @@ const COLUMN_DATA: Column[] = [
   },
 ];
 
-const AuditLogsTable = ({ showModel = false }: Props) => {
+const AuditLogsTable = () => {
   const { modelName } = useParams<EntityDetailsRoute>();
+  const showModel = !modelName;
```

Comment:
Instead of having `showModel` props I've changed these components to instead check if there is a model name in the current route.